modules/howtos/examples/managing_connections.py

```diff
@@ -1,11 +1,20 @@
+# requires:
+#  - a local couchbase server with
+#      username/password
+#      travel-sample
+#      beer-sample
+#  - /etc/hosts (or C:\Windows\System32\Drivers\etc\hosts on Windows)
+#      with `127.0.0.1 node2.example.com`
+
 from couchbase.cluster import Cluster, ClusterOptions
 from couchbase_core.cluster import PasswordAuthenticator
 
 class ManagingConnections(object):
     def test_simpleconnect(self):
 
+      print("simpleconnect")
       #tag::simpleconnect[]
-      cluster = Cluster.connect("127.0.0.1", ClusterOptions(PasswordAuthenticator("username", "password")))
+      cluster = Cluster.connect("couchbase://127.0.0.1", ClusterOptions(PasswordAuthenticator("username", "password")))
```

Comment:
Should we not use `localhost` here? And in a few other places below I think.

Reply:
ah yes, we did agree that didn't we? will update